Mathlib/Order/Fin/Basic.lean

```diff
@@ -337,6 +337,14 @@ lemma rev_anti : Antitone (@rev n) := rev_strictAnti.antitone
 @[simps! apply]
 def valOrderEmb (n) : Fin n ↪o ℕ := ⟨valEmbedding, Iff.rfl⟩
 
+namespace OrderEmbedding
+
+@[simps]
```

Comment:
I think this generates the name `OrderEmbedding.default_def`, which doesn't mention `Fin` or `Nat`